exercises/practice/rtos-static-scheduling/.meta/config.json

```diff
@@ -0,0 +1,22 @@
+{
+  "blurb": "Given a task list, determine a schedule in which the tasks can execute sequentially with no overlap",
+  "authors": [
+    "CSwift8"
+  ],
+  "contributors": [
+    "Newby555"
+  ],
+  "files": {
+    "solution": [
+      "rtos_static_scheduling_example.py"
+    ],
+    "test": [
+      "rtos_static_scheduling_test.py"
+    ],
+    "example": [
+      "rtos_static_scheduling.py"
+    ]
+  },
```

Comment:
@Newby555 should I put rtos_static_scheduling_classes.py in these "files"?  It needs to be visible to the user so in "solution"?

Reply:
Yes you can do that